assets/css/index.css

```
@@ -23,18 +23,30 @@ section.prose {
   @apply font-geist
 }
 
-.prose h1,
-.prose h3,
-.prose h4,
-.prose h5,
-.prose h6 {
-  @apply font-normal;
+.prose h1 {
+  @apply text-4xl font-normal break-words;
 }
 
 .prose h2 {
   @apply text-lg font-medium pb-3 border-b border-b-redis-pen-700 border-opacity-50;
```

This is the possible change I mentioned during our meeting earlier. It should increase the size of h2 to fall between h1 and h3.
```suggestion
  @apply text-2xl font-medium pb-3 border-b border-b-redis-pen-700 border-opacity-50;
```